pom.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.github.rodionovsasha</groupId>
+    <artifactId>commentservice</artifactId>
+    <version>1.0.0</version>
+    <packaging>jar</packaging>
+
+    <name>CommentService</name>
+    <description>Demo project for JFixtures</description>
+    <url>https://github.com/rodionovsasha/comment-service</url>
+
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>http://www.opensource.org/licenses/mit-license.php</url>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Aleksandr Rodionov</name>
+            <email>allrod@mail.ru</email>
+            <organizationUrl>https://github.com/rodionovsasha</organizationUrl>
+        </developer>
+        <developer>
+            <name>Vladimir Korobkov</name>
+            <email>vladimir.korobkov@gmail.com</email>
+            <organizationUrl>https://github.com/vkorobkov</organizationUrl>
+        </developer>
+    </developers>
+
+    <scm>
+        <connection>scm:git:git://github.com/vkorobkov/comment-service.git</connection>
+        <developerConnection>scm:git:ssh://github.com:vkorobkov/comment-service.git</developerConnection>
+        <url>https://github.com/rodionovsasha/comment-service</url>
+    </scm>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>1.5.6.RELEASE</version>
+        <relativePath/>
+    </parent>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <java.version>1.8</java.version>
+        <jfixtures.version>1.0.21</jfixtures.version>
+        <swagger.version>2.7.0</swagger.version>
```

Comment:
Do we really need swagger for this small demo project?

Reply:
hm, it is convenient to see API docs.